### PR TITLE
ZURI CHAT - My PR fixes Manage your Workspace page Link and route properly

### PR DESCRIPTION
**Pull request by Chisquare7**
Linear Ticket link: https://linear.app/team-brainbox/issue/FRO-74/zuri-chat-fix-the-manage-your-workspace-page-link-and-route-properly

# Fixes Issue/Linear Ticket
**My PR Fixes ID-linear_ticket_number_FRO-74**

# Changes proposed
Worked on fixing the Manage your Workspace page link and route properly

## What were you told to do?
ZURI CHAT-Fix the manage your workspace page Link and route properly

## What did you do?
- I moved AdminSettings files from the old src folder to new src folder in the protected folder and changed the file name to ManageWorkspace.jsx
- I fixed all the import errors and paths of other files associated with ManageWorkspace.jsx
- I fixed the routing links for ManageWorkspace file in the App.jsx file in the new src folder
- I moved necessary and equivalent assets from old src folder to new src folder

# Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/ Videos

Before changes:
manage-ur-workspace

After changes:
image

### CODE REVIEW

File: packages/main/src/components/display-pronoun/DisplayPronoun.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  btn: {
+    backgroundColor: "#00B87C",
+    color: "#ffffff",
+    border: "none"
+  }
+};
+
+const modifyOrganizationSettings = () => {
+  let organizationid = "6158c1ac3a1e49b1e5165df3";
```
Comment (JulianaSau) [inline packages/main/src/components/display-pronoun/DisplayPronoun.jsx:19]: Why is the org id hardcoded?😅

File: packages/main/src/components/workspace-language/WorkspaceLanguage.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  btn: {
+    backgroundColor: "#00B87C",
+    color: "#ffffff",
+    border: "none"
+  }
+};
+
+const modifyOrganizationSettings = () => {
+  let organizationid = "6158c1ac3a1e49b1e5165df3";
+  let workspacelanguage = "";
```
Comment (JulianaSau) [inline packages/main/src/components/workspace-language/WorkspaceLanguage.jsx:18]: Same here. If you have it hardcoded anywhere else in the admin settings pages please fetch it dynamically. I believe it's usually stored in localStorage

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/workspace-language/WorkspaceLanguage.jsx (hunk 2 above)
Comment (Chisquare7) [reply packages/main/src/components/workspace-language/WorkspaceLanguage.jsx:18]: Request changed and updated

File: packages/main/src/components/display-pronoun/DisplayPronoun.jsx (hunk 1 above)
Comment (Chisquare7) [reply packages/main/src/components/display-pronoun/DisplayPronoun.jsx]: Request changed and updated

**Comment by Chisquare7** [comment]
Request changed and updated

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**